starboard/nplb/posix_compliance/posix_epoll_create_test.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fcntl.h>
+#include <sys/epoll.h>
+#include <unistd.h>
+
+#include "starboard/nplb/posix_compliance/posix_epoll_test_helper.cc"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace starboard {
+namespace nplb {
+namespace {
+
+// --- Tests for epoll_create1 ---
+// epoll_create1 is epoll_create without the obsolete size argument.
+class PosixEpollCreate1Tests : public PosixEpollTest {};
+
+TEST_F(PosixEpollCreate1Tests, CreateSuccessfully) {
+  int epfd = epoll_create1(0);
+  ASSERT_NE(epfd, -1) << "epoll_create1(0) failed: " << strerror(errno);
+  if (epfd != -1) {
+    ASSERT_EQ(close(epfd), 0);
```

Comment:
`close()` also sets errno when failing (`close() returns zero 
on success. On error, -1 is returned, and errno is set 
appropriately.`). So this should be
```
    ASSERT_EQ(close(epfd), 0) << "close: " << strerror(errno);
```